selfdrive/ui/qt/util.h

```diff
@@ -1,15 +1,18 @@
 #pragma once
 
-#include <QDateTime>
-#include <QLayout>
-#include <QLayoutItem>
-#include <QMouseEvent>
-#include <QPainter>
 #include <QSurfaceFormat>
 #include <QWidget>
 
 #include "selfdrive/common/params.h"
 
+class QDateTime;
+class QLayout;
+class QMouseEvent;
+class QPainter;
```

Comment:
use forward declaration for classes can significantly reduce build times. the Qt's source code and examples use  forward declarations everywhere.so I think this may also be one of qt best practices

Reply:
Good to know they to this. Let's stick to the header files for consistency now. 

If we do the forward includes we should do it across all the Qt code (after verifying it has a significant impact on build time).